package.json

```diff
@@ -45,6 +45,8 @@
     "pump": "^1.0.0",
     "pumpify": "^1.3.3",
     "relative-date": "^1.1.2",
+    "request": "^2.58.0",
```

Comment:
request is quite a heavy dependency... I think we should try using `require('got')` instead